core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -143,21 +143,21 @@ class AdminClient(val time: Time,
                              clientHost: String,
                              assignment: List[TopicPartition])
 
-  def describeConsumerGroup(groupId: String): List[ConsumerSummary] = {
+  def describeConsumerGroup(groupId: String): Option[List[ConsumerSummary]] = {
```

Comment:
To me it seems like this bug is fixing a corner case so the chances of the rebalancing state of a consumer group being important information later on are low. Though I'm very new to kafka so I could be completely missing the point here.